include/structs.h

```diff
@@ -941,20 +941,28 @@ typedef struct unk80027FC4 {
     s16 unk6;
 } unk80027FC4;
 
-// Unused
-typedef struct GhostHeaderChecksum {
-	u8  levelID;
-    u8  vehicleID; // 0 = Car, 1 = Hovercraft, 2 = Plane
-} GhostHeaderChecksum;
+typedef struct GhostLocationVehicle {
+	/* 0x00 */ u8  levelID;
+    /* 0x01 */ u8  vehicleID; // 0 = Car, 1 = Hovercraft, 2 = Plane
+} GhostLocationVehicle;
+
+typedef struct GhostCharacter {
+    /* 0x02 */ u8  ID; // 9 = T.T.
+    /* 0x03 */ u8  unk3; // Might just be padding?
```

Comment:
Yeah, I should probably revert my changes to this struct, I think this might be two distinct structs entirely, and trying to force union them together is messy.